README.md

```diff
@@ -1,77 +1,24 @@
 <!--
    - SPDX-FileCopyrightText: 2019-2021 Serokell <https://serokell.io>
    -
-   - SPDX-License-Identifier: LicenseRef-ReplaceMe
+   - SPDX-License-Identifier: Apache-2.0
    -->
 
-**Please read this file as a raw textual file, it has comments for people
-who create a new repo from this template.**
+[![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-brightgreen.svg)](https://opensource.org/licenses/Apache-2.0)
 
-This repository is a template that we use at Serokell to create new repositories.
-See [GitHub documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template)
-for more details about this feature.
+# Customer.io client for haskell
```

Comment:
Let's have a bit more meaningful section, we should probably mention that this client is `servant-client` based and currently supports only some subset of `Track` customer.io API